src/Components/CoinData.jsx

```diff
@@ -1,58 +1,81 @@
-import { useState, useEffect, useContext} from 'react'
-import axios from 'axios'
+import { useState, useEffect, useContext } from "react";
+import axios from "axios";
 
-import { SearchContext } from './Contexts/searchContext';
-
-import Backdrop from '@mui/material/Backdrop';
-import Card from '@mui/material/Card';
-import CardHeader from '@mui/material/CardHeader';
-import CardMedia from '@mui/material/CardMedia';
-import CardContent from '@mui/material/CardContent';
-import CardActions from '@mui/material/CardActions';
-import Typography from '@mui/material/Typography';
+import { SearchContext } from "./Contexts/searchContext";
```

Comment:
First external imports occur that is imports from installed libraries, then 1 line space, then all internal imports that is files/components/functions that you have created, then 1 line space and then finally all css imports.